### ISSUE TITLE
Wrong OpenVINO installation on Windows host

### ISSUE THREAD

**Issue by newmluser** [opencv/cvat#534]
I installed the latest version on windows (using docket mentioned).
I do not see the Analytics and model manager option? how can I get them,

based on the documentation I have installed in windows already from the link.
https://docs.openvinotoolkit.org/2019_R1/_docs_install_guides_installing_openvino_windows.html

am I missing anything

**Comment by bsekachev** [comment]
OpenVino can't be installed. 
There is the error 
```
" is not a valid value for "ACCEPT_EULA"
```

Probably it is connected with different approaches to process line endings in Linux and Windows.

**Comment by bsekachev** [comment]
Related PR: #540

**Comment by bsekachev** [comment]
Commands below setup line endings corresponding to updated .gitattributes in PR: #540 
```
# Please be sure, that your working tree is empty
git rm --cached -r .
git reset --hard
```

You also can remove all repository and clone it again.

**Comment by bsekachev** [comment]
Fix merged in develop #540
